### PR TITLE
Updated trainer to accept training data in either in either TSV format or TFRecords format

### PR DESCRIPTION
**Pull request by nkashy1**
Tests updated.

They only cover the TSV case for now, as the TFRecords schema will change to having all columns be numeric.

### CODE REVIEW

**Comment by nkashy1** [comment]
Most recent commit uses tf.contrib.learn.learn_runner to distribute the task over a cluster. The previous version handled distribution badly.

**Comment by waprin** [comment]
Prob better to do as one big separate followup PR, but both PEP8 and Google Python style guide are 4 space tabs.

File: spark-tensorflow/trainer/data.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import multiprocessing
+import tensorflow as tf
+
+TSV = 'tsv'
```
Comment (waprin) [inline spark-tensorflow/trainer/data.py]: Why do we keep calling it TSV instead of CSV?

**Comment by waprin** [comment]
Yeah a bunch more style nits but the overall code LGTM so just going to merge.

File: spark-tensorflow/trainer/data.py (hunk 1 above)
Comment (nkashy1) [reply spark-tensorflow/trainer/data.py]: (Discussed in person)

Tab-separated values

**Comment by nkashy1** [comment]
@waprin : Added [this issue](https://github.com/GoogleCloudPlatform/cloud-dataproc/issues/15), will take it up after merge.

**Pull request merged**